docs/api.md

```diff
@@ -794,6 +795,45 @@ Enables Dark Mode support on macOS 10.14+.
 
 See https://electronjs.org/docs/api/browser-window#setting-backgroundcolor
 
+#### [bookmarks-menu]
+
+```
+--bookmarks-menu <string>
+```
+
+Path to a JSON file containing configuration for a predefined bookmarks menu. In addition to containing a list of bookmarks, this file customizes the name of the menu and (optionally) allows assigning keyboard shortcuts to bookmarks.
```

Comment:
Concision nit.

Also, let's mention that folders are not supported

Also, let's mention the syntax to assign a `Alt + something` shortcut.

```suggestion
Path to a JSON file defining a bookmarks menu. In addition to containing a list of bookmarks, this file customizes the name of the menu and (optionally) allows assigning keyboard shortcuts to bookmarks.

This menu is a simple list; folders are not supported.

Your `menuLabel` can be bound to a `Alt + letter` shortcut using the letter `&` before the `letter` you want. Be careful to not conflict with the letter of other menus!
```